tools/experiments/experiments.js

```diff
@@ -138,6 +143,34 @@ function build() {
   } else {
     redirect.setAttribute('hidden', '');
   }
+
+  const rtvInput = document.getElementById('rtv');
+  const rtvButton = document.getElementById('rtv-submit');
+  rtvInput.addEventListener('input', () => {
+    rtvButton.disabled = !(
+      rtvInput.value == '' || RTV_PATTERN.test(rtvInput.value)
+    );
+    rtvButton.textContent = rtvInput.value == '' ? 'opt-out' : 'opt-in';
```

Comment:
Nit: this can just be `rtvInput ? 'opt-in' : 'opt-out'`

Reply:
No, but I can do `rtvInput.value ? 'opt-in' : 'opt-out';` :)